meson.build

```diff
@@ -0,0 +1,819 @@
+# TODO: cross-compile supports is rudimentary
+
+project('uftrace', 'c', version: '0.9.4')
+python_module = import('python')
+
+common_cflags = ['-D_GNU_SOURCE', '-D_DEFAULT_SOURCE', '-D_XOPEN_SOURCE=600']
+common_cflags += '-fvisibility=hidden'
+common_cflags += '-fno-omit-frame-pointer'
```

Comment:
Actually these two flags are for libmcount.

Common flags should be: -W -Wall -Wdeclaration-after-statement -Wno-unused-parameter -Wno-missing-field-initializers